Dockerfile-base

```diff
@@ -0,0 +1,25 @@
+# This file creates a "studentenportal-base" image with all studentenportal
+#  dependencies.
+FROM python:2.7
+
+#TODO: Do we really need build-essential here?
+RUN apt-get update && \
+  apt-get -y install \
+    build-essential gcc g++ \
+    python-dev python-setuptools python-pip python-virtualenv \
+    git mercurial subversion \
+    postgres-client libpq-dev \
+    libjpg-progs libjpg-dev imagemagick \
+    ruby ruby-dev \
```

Comment:
Running the docker build of the current Dockerfile fails installing standard packages
```
...
Get:7 http://deb.debian.org jessie/main amd64 Packages [9064 kB]
Fetched 9848 kB in 31s (311 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package postgres-client
E: Unable to locate package libjpg-progs
E: Unable to locate package libjpg-dev
```
Any suggestions on wha I am missing here?